hypervisor/vm.go

```diff
@@ -574,6 +632,82 @@ func (vm *Vm) RemoveVolumes(names ...string) (bool, map[string]api.Result) {
 	return vm.batchWaitResult(names, vm.ctx.RemoveVolume)
 }
 
+func (vm *Vm) GetIPVSService() ([]byte, error) {
+	cmd := []string{"ipvsadm", "-Ln"}
+	res, err := vm.hyperstartExecSync(cmd, nil)
+	if err != nil {
+		return nil, err
+	}
+	if res.code != 0 {
+		err = fmt.Errorf("get ipvs service from vm failed: %d, %s", res.code, res.stderr)
+		glog.Error(err)
+		return nil, err
+	}
+
+	return res.stdout, nil
+}
+
+func (vm *Vm) ApplyIPVSService(patch []byte, clearFirst bool) error {
```

Comment:
Do you mean inline like `sh -c "ipvsadm ....."` or pack them with hyperstart? It's could be like that, just not sure why we need do that, because this seems not a complicated logic.